### PR TITLE
Remove MAS-SSO token retrieval, configurations, and keycloak-js library

### PR DESCRIPTION
**Pull request by MikeEdgar**
Note: the JSON entries in `config.json` must remain to pass schema validation, but the values have been set to be clear that they are not used. These changes work for me locally with kas-installer and production SSO.

### CODE REVIEW

File: config/config.json
Comment (ajaypratap003) [inline config/config.json:63]: I think we should remove masSso object as we don't need it anymore.

**Review by ajaypratap003 — COMMENTED** [review]
Changes looks good to me. 
Thanks for the PR.

**Comment by ajaypratap003** [comment]
@MikeEdgar There are some  changes need to clean and I already did on my machine. I can push into your PR if you don't mind. 
Please give access on this PR's branch so that I can push my changes. 
Github ID: ajaypratap003

**Comment by MikeEdgar** [comment]
@ajaypratap003 , please go ahead. I have 'allow edits by maintainers' enabled so you should be able to push to this PR's branch in my fork.

File: config/config.json
Comment (MikeEdgar) [reply config/config.json:63]: I agree and I tried initially, but it failed to loading here. I think it first needs to be removed from the `Config` in `@rhoas/app-services-ui-shared`.
https://github.com/redhat-developer/app-services-ui/blob/642cfdfa44f60fc129fc15a411fb376274104505/src/app/providers/config/EmbeddedConfigContextProvider.tsx#L15-L16

**Comment by riccardo-forina** [comment]
I'm not so sure we can remove keycloak completely. 

We can run without keycloak on consoledot as they'll provide and handle the auth for us, but what about when the UI is running against a fully local stack, with its own SSO? Keycloak will be needed in that case to handle the refresh of the token in the least.

**Comment by MikeEdgar** [comment]
> We can run without keycloak on consoledot as they'll provide and handle the auth for us, but what about when the UI is running against a fully local stack, with its own SSO? Keycloak will be needed in that case to handle the refresh of the token in the least.

@riccardo-forina can we confirm that? I believe the Keycloak code removed here was only used for the MAS-SSO token retrieval.

**Comment by ajaypratap003** [comment]
> @ajaypratap003 , please go ahead. I have 'allow edits by maintainers' enabled so you should be able to push to this PR's branch in my fork.

I've pushed my changes and also  removed mas-sso object from config. 
PR is open for ui-shared https://github.com/redhat-developer/app-services-ui-shared/pull/127
I'll update package in this PR once ui-shared PR will merge.

**Review by ajaypratap003 — APPROVED** [review]

**Pull request merged**
